File: python/Prostokat.py

def main(args):
    bok1 = int(input("Podaj 1 bok: "))
    bok2 = int(input("Podaj 2 bok: "))
    pole = bok1 * bok2
    print("Pole: ", pole)
    obwod = 2 * bok1 + 2 * bok2
    print("Obwod: ", obwod)
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: C++/Programy/prostokat.cpp

#include <iostream>
using namespace std;
int main(int argc, char **argv)
{
	int bok1;
	int bok2;
	int pole;
	int obwod;
	bok1 = 0;
	bok2 = 0;
	pole = 0;
	cout << "Podaj pierwszy bok prostokąta: ";
	cin >> bok1;
	cout << "Podaj drugi bok prostokąta: ";
	cin >> bok2;
	pole = bok1 * bok2;
	cout << "Pole prostokąta: " << pole << endl;
	obwod = bok1 * 2 + bok2 * 2;
	cout << "Obwód: " << obwod;
	
	return 0;
}

